include/inexor/vulkan-renderer/wrapper/command_buffer.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "inexor/vulkan-renderer/wrapper/resource_descriptor.hpp"
```

Comment:
This can be replaced with a forward declaration, like with the old `Descriptor`

Reply:
Done.